### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by NoahSchiro**
Added metrics for transformers and opened the transformer model up to the whole dataset, earning us a accuracy of 80.7%

accuracy
loss

### CODE REVIEW

**Review by Kofwana — APPROVED** [review]
Fantastic job on bringing metrics to the transformers! Now, our models can strut their stuff across the entire dataset, flaunting an impressive 80.7% accuracy. It's like giving our models a VIP pass to the data party! 🚀 Keep up the awesome work – our transformers are feeling snazzy with these new moves! 👏😄

**Review by azizakaa — APPROVED** [review]
These metrics look really good, good job Noah!

**Comment by Kofwana** [comment]
I've pushed my updated code to this branch and I believe it shows up in this PR including the updated graphs. This added code in train.rnn calculates and stores the accuracy, training loss, and validation loss for each epoch. After training, it plots two graphs: one for accuracy vs epochs and another for training and validation loss vs epochs. Unfortunately we cannot include them in the report but we could consider adding them to the readme and referencing that in the report. (thankfully there doesn't seem to be any merge conflicts here)
